src/ServiceManagement/Network/Commands.Network/Gateway/NewAzureVirtualNetworkGatewayConnection.cs

```diff
@@ -32,8 +32,9 @@ public class NewAzureVirtualNetworkGatewayConnectionCommand : NetworkCmdletBase
         [ValidateNotNullOrEmpty]
         public string GatewayConnectionName { get; set; }
 
-        [Parameter(Position = 2, Mandatory = true, HelpMessage = "Gateway connection type: Ipsec/Dedicated/VpnClient/Vnet2Vnet")]
+        [Parameter(Position = 2, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Gateway connection type.")]
         [ValidateNotNullOrEmpty]
+        [ValidateSet(ServiceManagement.Model.VirtualNetworkGatewayConnectionType.ExpressRoute, ServiceManagement.Model.VirtualNetworkGatewayConnectionType.IPsec, ServiceManagement.Model.VirtualNetworkGatewayConnectionType.Vnet2Vnet, ServiceManagement.Model.VirtualNetworkGatewayConnectionType.VPNClient, IgnoreCase = true)]
```

Comment:
@Nilambari this is a breaking change; existing scripts that use this parameter with any value that's not in the ValidateSet will no longer work